config/app.default.php

```diff
@@ -236,7 +236,7 @@
             'timezone' => 'UTC',
             'flags' => [],
             'cacheMetadata' => true,
-            'log' => false,
+            'log' => true,
```

Comment:
Well the intention was to help with getting people going with development and discovering the query log.

In a similar way wouldn't you say we need to disable 4xx logging, then, too?

If reading the debug flag was available here... I'd say set it to true on debug, false on debug false?